src/ConfigManager.cpp

```diff
@@ -434,16 +434,26 @@ void ConfigManager::createBaseWebServer() {
 void ConfigManager::streamFile(const char* file, const char mime[]) {
   SPIFFS.begin();
 
+  size_t len = strlen(file);
+  char* path = new char[len+1];
```

Comment:
with the `delete` L456, if this wasn't declared outside the IF, had an error ... scoping and C++ is a mystery